CalorieTracker/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '9.0'
```

Comment:
Usually we would want to change this value to the lowest OS version you're going to support.